docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-overview.md

```diff
@@ -113,6 +125,12 @@ If you need a feature from a preview release (that isn't yet available in a stab
 
 Yugabyte manages upgrades for you. After you choose a track, database upgrades continue to take releases from the track you chose. For multi-node clusters, Yugabyte performs a rolling upgrade without any downtime. You can manage when Yugabyte performs maintenance and upgrades by configuring the [maintenance window](../../cloud-clusters/cloud-maintenance/) for your cluster.
 
+#### Staging clusters
+
+Yugabyte tests every version in the stable branch for backwards compatibility. However, it's good practice to first test database updates against your pre-production environment (aka development, testing, staging, or canary environment) to ensure compatibility before upgrading your production clusters.
+
+Create a staging cluster (this can be smaller than your production cluster) and configure your pre-production environment to connect to it. Be sure to configure the [maintenance windows](../../cloud-clusters/cloud-maintenance/) for the staging and production cluster so that you can validate updates against your applications in your pre-production environment before updating your production cluster.
+
```

Comment:
I don't think this is really possible yet in a bullet-proof way. For example while a customer may set their maintenance window for staging cluster on Mondays and production cluster on Wednesdays, if we schedule the upgrade on the Tuesday, the prod cluster will still have maintenance scheduled first. We have a design internally to implement maintenance priority of clusters within the same account, but we probably won't get around to implementing it for a few more months.

Reply:
Agreed, but we could mitigate that with an internal policy of scheduling maintenance windows based on a weekly cycle.